README.md

```diff
@@ -24,7 +24,9 @@ It also includes an End-to-End test.
 
 ## How to run locally in development
 
-To test out a local build of the Snooty VSCode Extension, run the command `vsce package` at the root. Once successfully compiled, open a Docs Content repo (e.g. `cloud-docs`, `docs-landing`) in a VSCode window and navigate to the `Extensions` panel from the lefthand `Extensions` sidebar button. Click the button at the top of the panel with the ellipses (...) and select `Install from VSIX`. Choose the newly compiled file from your local `snooty-vscode` repo named in the format of `snooty-<version>.vsix`. This should enable your local branch as the extension utilized on your local machine's VSCode.
+To test out a local build of the Snooty VSCode Extension, you first need to run `npm install -g @vscode/vsce` to install the `vsce` package globally. Then, follow the instructions in [`HACKING.md`](https://github.com/mongodb/snooty-vscode/blob/main/HACKING.md#setting-up-parser-communication) to allow the extension to communicate with the parser locally.
+
+Run the command `vsce package` at the root of this directory to compile the project. Once successfully compiled, open a Docs Content repo (e.g. `cloud-docs`, `docs-landing`) in a VSCode window and navigate to the `Extensions` panel from the lefthand `Extensions` sidebar button. Click the button at the top of the panel with the ellipses (...) and select `Install from VSIX`. Choose the newly compiled file from your local `snooty-vscode` repo named in the format of `snooty-<version>.vsix`. This should enable your local branch as the extension utilized on your local machine's VSCode.
```

Comment:
I'm not sure - I'm not a big function key user, but when testing it seems like this method doesn't work:
<img width="298" alt="image" src="https://github.com/mongodb/snooty-vscode/assets/41043008/6f7c03cc-401d-4914-a976-2aff6f2c7dd4">

Reply:
We should absolutely fix the issue preventing the debug button from working. It's a small one line fix https://jira.mongodb.org/browse/DOP-4705

Reply:
Gotcha, thanks! Idk why it was working on my end when I tried last week. 🤔 

I think it might be confusing to have 2 different sections on how to run the extension locally in 2 different places in the repo. It might be good to consolidate the instructions with clarity on the differences, but that could always be included in the above ticket